### ISSUE TITLE
[Bug]: `step certificate inspect` no longer supports reading cert from piped out put

### ISSUE THREAD

**Issue by a1994sc** [smallstep/cli#1173]
### Steps to Reproduce

1. `step certificate inspect --format pem --insecure https://github.com > github.pem`
2. `step certificate inspect github.pem` This works
3. `cat github.pem | step certificate inspect` This does not

### Your Environment

* OS - Fedora 39
* `step` CLI Version - `0.26.1`

### Expected Behavior

Read the cert from the piped output

### Actual Behavior

gives `error reading -: no such file or directory`

### Additional Context

I have some certs in `sops` secrets and want to read the alternative names, so I run the following command with `0.26.1` and it errored out

```shell
sops -d secret/tls.yml | yq '.stringData."tls.crt"' | step certificate inspect --format json | yq -p=json .names | sort | yq
```

### Contributing

Vote on this issue by adding a 👍 reaction.
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by a1994sc** [comment]
Minor comment, but step 3. does work with `0.25.2`

**Comment by dopey** [comment]
Hey @a1994sc 👋. Thanks for opening the issue!

I believe this has already been resolved in the default branch - #1162. Should be part of a release within the next few weeks.

**Comment by a1994sc** [comment]
Oh, @dopey , sorry I must have missed that when I went looking for issues. Thank you very much! I will keep an eye out for it. For the time being I rolled back my version to `0.25.2`
